README.rst

```diff
@@ -65,9 +65,8 @@ extending the traditional cryptological narrative of "Alice and Bob" by introduc
 
 Features
 ==========
-- Re-encryption Toolkit
+- Proxy Re-encryption Toolkit
```

Comment:
I pondered including "Proxy" here, but since there is no network proxy tooling anywhere in pyUmbral (it's in the KMS), I omitted it to not be misleading; Nonetheless - your intention here is to indicate Umbral's *cryptographic*, not network notion of a proxy?

Reply:
OK I understand your intention, but since this is very decoupled from the KMS, I think there is not much risk of confusion.

Reply:
I don't know If I agree that there is not much risk of confusion (at least when examining pyUmbral from the perspective of trying understand the Umbral scheme and NuCypher holistically) since a network proxy is such an essential component of most obvious use cases. Perhaps any confusion people experience today is a side-effect of not having robust KMS docs published yet.

Reply:
Hmm, I think I'm with @KPrasch on this one.  The toolkit in question doesn't do anything that's particular to a *proxy*, right?  As opposed to re-encryption performed by some other agent?

Or is it customary to *always* regard anyone perform re-encryption as a "proxy"?